data/lib/core/storages.lua

```diff
@@ -2158,7 +2158,18 @@ Storage = {
 					SnakeHeadTeleport = 42731, -- 7th Tome: Snake Head Teleport
 					CorruptionHole = 42732 -- 8th Tome: Corruption Hole
 				},
-			TomesOfKnowledge = {}, -- 42751 - 42780
+			TomesOfKnowledge = {42751, -- storage for all tomes
+			42752,
+			42753, -- 3
+			42754,
+			42755,
+			42756, -- 6
+			42757,
+			42758,
+			42759, -- 9
+			42760,
+			42761,
+			42762}, -- 42751 - 42780
```

Comment:
We can improve this and keep it in the default file
```suggestion
			-- Reserved storages range 42751 - 42780
			TomesOfKnowledge = {
						Tome1 = 42751,
						Tome2 = 42752,
						Tome3 = 42753,
						Tome4 = 42754,
						Tome5 = 42755,
						Tome6 = 42756, -- 6
						Tome7 = 42757,
						Tome8 = 42758,
						Tome9 = 42759, -- 9
						Tome10 = 42760,
						Tome11 = 42761,
						Tome12 = 42762
			},
```